gemet/thesaurus/views.py

```diff
@@ -260,7 +260,7 @@ def get_object(self):
             .values('language__name', 'value')
         )
         concept.set_attributes(self.langcode, self.attributes)
-        concept.url = settings.GEMET_URL + concept.get_about_url()
+        concept.url = self.model.objects.get_ns().url + code
```

Comment:
I think `concept.namespace.url` is clearer than `self.model.objects.get_ns().url`